poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
Not this file. You need to navigate inside the connector folder and run inside it.
```
cd airbyte-integrations/connectors/source-slack
poetry lock
```
Please revert this change and follow the steps above.

Reply:
Oh shoot, my bad. Fixing that now

Reply:
Fixed @marcosmarxm. Thanks for catching that